content/queue/queue.go

```diff
@@ -140,7 +140,7 @@ func (qm *queueManager) processQueue() {
 
 func (qm *queueManager) ToCheck(contID uint64, contSize int64) {
 	// if DisableFilecoinStorage is not enabled, queue content for deal making
-	if !qm.isDisabled && contSize >= qm.dealSizeLimit {
```

Comment:
Any reason for removing the size check? The reason for the check is that we dont want to add content that is not ready for deal-making into the deal queue. Why queue them if they will be ignored downstream? This will only make the deal queue grow larger